Library/Homebrew/gpg.rb

```diff
@@ -7,7 +7,7 @@ def self.find_gpg(executable)
       next unless gpg_short_version
       gpg_version = Version.create(gpg_short_version.to_s)
       @version = gpg_version
-      gpg_version == Version.create("2.1") ||
+      gpg_version >= Version.create("2.1") ||
```

Comment:
Any reason why we don't want `gpg_version >= Version.create("2.0")`?

Reply:
That would work too, aye. I was partially thinking in the future that `2.0` is going to go away anyway because upstream are dropping support for it, but yeah, there's no actual reason to wait to compact it down to one line.